data-sources/gigamon-appinsights/config.yml

```diff
@@ -0,0 +1,28 @@
+id: gigamon-appinsights
+displayName: Gigamon Application Insights
+description: |
+  Gigamon Application Intelligence gives NetOps, CloudOps, and security teams full visibility into applications running in datacenters and public clouds. Therefore, you can maintain consistently strong security across your environment plus get to the root causes of vulnerabilities and performance bottlenecks.
+icon: logo.png
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId: d6b36ecf-bf99-475c-938f-370153db8e70
+      requiresAccount: true
+  fallback:
+    link:
+      url: https://www.gigamon.com/products/optimize-traffic/application-intelligence.html
+keywords:
+  - Gigamon
+  - Gigamon Application Intelligence
+  - Gigamon Integration
+  - featured
+  - newrelic partner
+  - security
+  - packet capture
+  - network
+  - NR1_addData
+  - NR1_sys
```

Comment:
We can remove a few of these since we are going to put them on the quickstart instead!
```suggestion
  - security
  - packet capture
  - network
```